### PR TITLE
stm32mp1: fix fuse programming support

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

File: core/arch/arm/plat-stm32mp1/nsec-service/bsec_svc.c [hunk 1]
```diff
@@ -32,11 +32,17 @@ void bsec_main(struct thread_smc_args *args)
...
+		else
+			DMSG("OTP programming not supported");
 		break;
 	case STM32_SIP_SVC_BSEC_WRITE_SHADOW:
 		FMSG("write shadow @%#"PRIx32, otp_id);
-		result = stm32_bsec_write_otp(in_value, otp_id);
+		if (IS_ENABLED(CFG_STM32_BSEC_WRITE))
+			result = stm32_bsec_write_otp(in_value, otp_id);
+		else
+			DMSG("OTP programming not supported");
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-stm32mp1/nsec-service/bsec_svc.c:45]: Wouldn't it be easier to keep the old inline function and let that print this?

Comment (etienne-lms) [reply core/arch/arm/plat-stm32mp1/nsec-service/bsec_svc.c:45]: Ok, I think this last commit is useless.
I'll discard it.
Thanks for the feedback.

**Comment by etienne-lms** [comment]
Updated

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jforissier** [comment]
@etienne-lms can you please add the tag? Thanks.

**Comment by etienne-lms** [comment]
tag applied.
thanks.

**Comment by jforissier** [comment]
The IBART failure will be gone after rebasing.

**Pull request merged**